docker/wrapper.py

```diff
@@ -98,10 +115,13 @@ def fileURL(sample):
     return 'file://' + sample
 
 
-def getSampleName(sample):
-    name = os.path.basename(sample).split('.')[0]
-    if name.endswith('R1') or name.endswith('R2'):
-        return name[:-2]
+def getSampleName(sample, output_basename):
+    if output_basename:
+        name = output_basename
+    else:
+        name = os.path.basename(sample).split('.')[0]
```

Comment:
Either this function needs to go and `output_basename` becomes a required variable (recommended), or you improve the heuristic for deriving a name form the sample string.  Filenames for samples can have dots in them that are not just for the extension.